code/datums/components/religious_tool.dm

```diff
@@ -242,6 +241,9 @@
 	if(!can_i_see)
 		return
 	examine_list += ("<span class='notice'>Use a bible to interact with this.</span>")
+	if(user.mind?.holy_role != NONE)
```

Comment:
Is this meant to be none, or a null check? Isn't none for bitflag checks?

Reply:
So yes, it was meant to be NONE because its default value is NONE